extensions/wikia/Search/classes/MediaWikiService.php

```diff
@@ -160,60 +180,76 @@ public function getSimpleLanguageCode() {
 
 	/**
 	 * Allows us to abstract out the MW Api for parse responses
+	 *
 	 * @param int $pageId
+	 *
 	 * @return array
 	 */
 	public function getParseResponseFromPageId( $pageId ) {
-		return \ApiService::call( array(
-					'pageid'	=> $pageId,
-					'action'	=> 'parse',
-				));
+		return \ApiService::call(
+			[
+				'pageid' => $pageId,
+				'action' => 'parse',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "action" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)